content/doc/book/blueocean/create-pipeline.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: section
+title: Creating Pipelines in Blue Ocean
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Creating Pipelines in Blue Ocean
+
+Blue Ocean makes it easy to create new Pipelines or create Pipelines from an existing
```

Comment:
The phrasing is awkward here with the subject of "creating a Pipeline". What about something like:

> Blue Ocean makes it easy to create a new Pipeline directly by importing a `Jenkinsfile` from Git or GitHub, or by creating a `Jenkinsfijle` from scratch with the Pipeline Editor.


I think the remainder of the section might need to use similar verbiage, insofar that "importing" might be used for telling Blue Ocean to pull an existing `Jenkinsfile` and "creating" referring to the editor-based flow.

If you think those make sense, the subsequent sections would need to be reworked a little bit.